### PR TITLE
v1.0.0-rc.1 release updates

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,7 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-/.nerves
 /build.log
 /archive.log
+/.nerves
+/doc
```
Comment (mobileoverlord) [inline .gitignore]: does it matter that there is no new line at the end of this file?

Comment (fhunleth) [reply .gitignore]: No, but it's annoying. I'll fix.

Comment (mobileoverlord) [reply .gitignore]: I checked other systems that had this applied recently and it made its way there too :(

Comment (fhunleth) [reply .gitignore]: I'll fix the rpi0 for the next release. I use that as my base for synchronizing all of the other systems.

**Pull request closed without merging**